lib/mutant/reporter/hash/printer.rb

```diff
@@ -0,0 +1,52 @@
+module Mutant
+  class Reporter
+    class Hash
+      # Hash runner status printer base class
+      class Printer
```

Comment:
I revert that one, see: https://github.com/mbj/mutant/pull/479#issuecomment-155875500